mautic/mautic
# Support for date/month date text segment filters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 
| New feature/enhancement? (use the a.x branch)      | ✅ 
| Deprecations?                          | 
| BC breaks? (use the c.x branch)        | 
| Automated tests included?              | not yet
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/266
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 



#### Description:



#### Steps to test this PR:
This PR added nee date/month text string segment date filter

- day check current day to date/datetime field month
- month - check current month to date/datetime field month


1. Load up [this PR](https://mautibox.com)
2. Create custom date time field
3. Create contact with value
- if today is 19. 10. 2021
- then add to custom field 19.09.2020
Create another contact and add
- if today is 19. 10. 2021
- then add to custom field 07.19.2019
3. Create segment 1 with field to custom field equal = day
4. Create segment 2 with field to custom field equal = month
5. Rebuild segment
6. You should see first contact added to segment 1
7. You should see second contact added to segment 2




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10542"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
